src/main/java/frc/robot/commands/AutomaticShootCommand.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class AutomaticShootCommand extends CommandBase {
+    private final ShooterSubsystem shooterSubsystem;
+    private final double targetVel;
+    private final int ballNum;
+    private int totalShoot = 0;
+
+    public AutomaticShootCommand(double targetVel, int ballNum, ShooterSubsystem shooterSubsystem) {
```

Comment:
ballNum indicates a specific ball number to shoot, while numShots or numBalls indicates how many you're going to shoot